Edits to UG and DG (#263)
AY2425S2-CS2103T-T13-1/tp: docs/DeveloperGuide.md

**Extensions**

- 1a. The client being searched for does not exist.<br>

>     - 1a1. FitFlow shows the list the clients with the keyword on the app.<br>

Reviewer: ```suggestion
    - 1a1. FitFlow shows an empty list of clients on the app.<br>
```

      Use case ends.

> - 1b. The keyword to find the client is empty.<br>

Reviewer: ```suggestion
- 1b. The keyword to find the client is empty or contains invalid characters.<br>
```

Author: as AnWei mentioned, I dont think there will be cases that contains invalid characters

> - 1b. The keyword to find the client is empty.<br>

Reviewer 2: Actually, following up on Desean's suggestion, the find feature currently accepts all characters as the keyword so not sure if there will ever be a case where there will be an error message showing "invalid characters". The only instance where the error message will show up is if the keyword is blank.

Author: yep my intial is only when keyword is empty
